runner.py

```diff
@@ -779,13 +786,19 @@ def get_main_parser():
                         if args.noHist == False:
                             save(
                                 output,
-                                args.output.replace(
+                                coffeaoutput.replace(
                                     ".coffea", f"_{sindex}_{findex}.coffea"
                                 ),
                             )
     if not "lxplus" in args.executor:
         if args.noHist == False:
-            save(output, args.output)
+            save(output, coffeaoutput)
+    if args.isArray:
+        if args.overwrite and path.exists(outdir):
+            os.system(f"rm -r {outdir}")
+        os.system(f"mkdir -p {outdir}")
+        os.system(f"cp -r tmp/* {outdir}/")
+        os.system("rm -rf tmp")
```

Comment:
Since the array saving is indeed before the jobs finished, so creating/deleting directory not needed here.